src/app/request-investigation/request-investigation.component.scss

```
@@ -0,0 +1,142 @@
+@use 'sass:math';
+
+h2,
+h3 {
+    font-weight: 700;
+    line-height: math.div(27, 23);
+    color: #000000;
+    font-family: Helvetica;
+    font-style: normal;
+}
+
+h2 {
+    font-size: 24px;
```

```suggestion
    font-size: 24px;
    line-height: math.div(27, 24);
```
`line-height` and `font-size` should go together.